SMLNJ/SMLNJ.munki.recipe

```diff
@@ -48,7 +48,10 @@
       <string>Software Development</string>
       <key>description</key>
       <string>Standard ML of New Jersey (abbreviated SML/NJ) is a compiler for the Standard ML '97 programming language with associated libraries, tools and documentation.  Standard ML is a safe, modular, strict, functional, polymorphic programming language with compile-time type checking and type inference.</string>
-      <key>postinstall_script</key>
+      <key>preinstall_script</key>
+      <string>#!/bin/sh
+pkgutil --forget org.smlnj.x86.pkg </string> 
```

Comment:
full path to pkgutil please (and no space at the end)

Reply:
All done soz